src/pages/collection/[id].tsx

```diff
@@ -0,0 +1,98 @@
+import { ApiClient } from "@/api/http-common";
+import { BreadCrumbs } from "@/components/breadcrumbs/Breadcrumbs";
+import Layout from "@/components/layouts/Main";
+import { SubNav } from "@/components/nav/SubNav";
+import { MultiCol } from "@/components/panels/MultiCol";
+import { SingleCol } from "@/components/panels/SingleCol";
+import { Heading } from "@/components/typography/Heading";
+import { withEnvConfig } from "@/context/EnvConfig";
+import { TCollection, TTheme } from "@/types";
+import { getFeatureFlags } from "@/utils/featureFlags";
+import { GetServerSideProps, InferGetStaticPropsType } from "next";
+import { LuFileStack } from "react-icons/lu";
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
Question / quibble: do we want to standardise putting the SS functions above the component? Is this the advised standard?